bee-api/bee-rest-api/README.md

```diff
@@ -0,0 +1,3 @@
+# bee-api
+
+A REST API implementation for IOTA node software.
```

Comment:
I suggest writing: "The default REST API implementation for the IOTA Bee node software."